docs/kpod-attach.1.md

```diff
@@ -0,0 +1,30 @@
+% kpod(1) kpod-attach - See the output of pid 1 of a container or enter the container
+% Dan Walsh
+# kpod-attach "1" "September 2017" "kpod"
+
+## NAME
+kpod-attach - Attach to a running container
+
+## Description
+
+We chose not to implement the `attach` feature in `kpod` even though the upstream Docker
+project has it. The upstream project has had lots of issues with attaching to running processes.
```

Comment:
Perhaps:
'running process.' to 'running process that we did not want to replicate.  The `kpod exec` and 'kpod log` commands will offer you the same functionality far more dependably.'